pallets/phala/src/compute/stake_pool_v2.rs

```diff
@@ -802,6 +802,67 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[pallet::weight(0)]
+		#[frame_support::transactional]	
+		pub fn reset_iter_pos(origin: OriginFor<T>) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			base_pool::Pallet::<T>::ensure_migration_root(who)?;
+			StakepoolIterateStartPos::<T>::put(None::<u64>);
+			Ok(())
+		}
+
+		#[pallet::weight(0)]
+		#[frame_support::transactional]
+		pub fn migrate_stakepools(origin: OriginFor<T>, max_iterations: u32) -> DispatchResult {
```

Comment:
It's not the initial migration. Better to give it a name like "fix_missing_worker_lock()"

Reply:
done